sonar-flex-plugin/pom.xml

```diff
@@ -134,10 +134,12 @@
       <plugin>
         <groupId>org.sonarsource.sonar-packaging-maven-plugin</groupId>
         <artifactId>sonar-packaging-maven-plugin</artifactId>
+        <version>1.22.0.705</version>
```

Comment:
why do we need to specify the version here?

Reply:
to use the latest release, eventually we should update parent pom to include this automatically